### DISCUSSION TITLE
✨ Redlib

### DISCUSSION THREAD

**Discussion by tempofunktron** [runtipi/runtipi-appstore#3816] [Requests]
### Description

Another privacy-focused Reddit viewer like Teddit or Libreddit, seems to be less prone to API issues

### Documentation link

https://quay.io/repository/redlib/redlib

### Last application release & date

0.34.0 (June 2024)

### Application license

AGPL-3.0

### Source code repository link

https://quay.io/repository/redlib/redlib

### Docker image link

https://pull quay.io/redlib/redlib

### Other's

https://github.com/redlib-org/redlib

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by jasonpearce** [comment]
This appears to be a duplicate of https://github.com/runtipi/runtipi-appstore/discussions/3178. I recommend merging requests into one.
